string ctrl_input
string explore_find_status